test/e2e/scheduler_predicates.go

```diff
@@ -219,24 +219,25 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 
 		By(fmt.Sprintf("Starting additional %v Pods to fully saturate the cluster max pods and trying to start another one", podsNeededForSaturation))
 
-		framework.StartPods(c, podsNeededForSaturation, ns, "maxp", api.Pod{
-			TypeMeta: unversioned.TypeMeta{
-				Kind: "Pod",
-			},
-			ObjectMeta: api.ObjectMeta{
-				Name:   "",
-				Labels: map[string]string{"name": ""},
-			},
-			Spec: api.PodSpec{
-				Containers: []api.Container{
-					{
-						Name:  "",
-						Image: "gcr.io/google_containers/pause:2.0",
+		if podsNeededForSaturation > 0 {
```

Comment:
i gues we assume that any given cluster may have a variable # of pods running, so this can easily be 0 if its run on a busy cluster, right?  If *yes* please add this to a comment here.